kornia/augmentation/utils/param_validation.py

```diff
@@ -12,29 +12,29 @@ def _common_param_check(batch_size: int, same_on_batch: Optional[bool] = None):
 
 
 def _range_bound(
-    factor: Union[torch.Tensor, float, Tuple[float, float], List[float]],
+    factor: Union[Tensor, float, Tuple[float, float], List[float]],
     name: str,
     center: float = 0.0,
     bounds: Tuple[float, float] = (0, float('inf')),
     check: Optional[str] = 'joint',
-    device: torch.device = torch.device('cpu'),
-    dtype: torch.dtype = torch.get_default_dtype(),
-) -> torch.Tensor:
+    device: Device = None,
+    dtype: Dtype = None,
+) -> Tensor:
     r"""Check inputs and compute the corresponding factor bounds."""
-    if not isinstance(factor, (torch.Tensor)):
-        factor = torch.tensor(factor, device=device, dtype=dtype)
-    factor_bound: torch.Tensor
+    if not isinstance(factor, (Tensor)):
```

Comment:
```suggestion
    if not KORNIA_CHECK_TYPE(factor, Tensor):
```
improvement proposal (in subsequent PRs) for the logger api to return value /cc @ducha-aiki

Reply:
I'm uncertain if [type guard](https://peps.python.org/pep-0647/) is supported by JIT, but will be the case here. Something like:

```python
from typing import TypeGuard, TypeVar


T = TypeVar('T')

def KORNIA_CHECK_TYPE(v: object, t: T) -> TypeGuard[T]:
    return isinstance(v, t)

```

Reply:
We should start not considering jit, it’s not supported anymore